common/dynamicconfig/constants.go

```diff
@@ -713,6 +713,11 @@ This config is EXPERIMENTAL and may be changed or removed in a later release.`,
 		10,
 		`FrontendMaxBadBinaries is the max number of bad binaries in namespace config`,
 	)
+	FrontendMaskInternalOrUnknownErrors = NewNamespaceBoolSetting(
+		"frontend.maskInternalErrors",
```

Comment:
I think config name should match var name.

Reply:
done in the next rev